R/plot.R

```diff
@@ -554,11 +554,11 @@ plot_pairwise_comparison <- function(comparison_result,
 #' plot_pit(pit)
 #'
 #' # quantile-based pit
-#' pit <- pit(example_quantile,by = "model")
+#' pit <- pit(as_forecast(example_quantile), by = "model")
```

Comment:
Does this imply we want to save our examples as forecast objects?

Reply:
I _think_ not. To me it feels having the example data equal to the raw data is instructive to users. 

If we think that calling `as_forecast()` every time is too much of a hassle, then maybe that has broader implications 🤣 (I'm joking here - to me everything is fine as is)

Reply:
(I see the joke but this doesn;t represent real work usage as in the real world you would transform and allocate once and then use that object)

Reply:
Yes agree. I was thinking that for the examples it's good to see `as_forecast()` be called explicitly because that tells you what to do. 
Whereas if you just have the object there and then call the function on your own data and that fails that might be more confusing. On the other hand, the functions are very explicit and tell you that you should call `as_forecast()` so it might also be fine. 

I'm still leaning towards keeping the raw data.tables, but I think I could be persuaded either way. Shall I make an issue?

Reply:
I think that could be a good idea. I don't feel that strongly as it isn't mission critical.